library/Zend/Mvc/Service/ServiceListenerFactory.php

```diff
@@ -53,6 +53,7 @@ class ServiceListenerFactory implements FactoryInterface
             'DiStrictAbstractServiceFactory' => 'Zend\Mvc\Service\DiStrictAbstractServiceFactoryFactory',
             'FilterManager'                  => 'Zend\Mvc\Service\FilterManagerFactory',
             'FormElementManager'             => 'Zend\Mvc\Service\FormElementManagerFactory',
+            'Zend\Form\Annotation\FormAnnotationBuilder' => 'Zend\Mvc\Service\FormAnnotationBuilderFactory',
```

Comment:
Quoting @Ocramius on why it was changed to the fqcn:

"I'd prefer to just have the FQCN here. Some advantages

    less keys to remember
    can be statically referenced via PHP 5.5's ClassName::class syntax, allowing IDE static introspection to find non-existing classes
    less documentation required
    more predictable from a consumer perspective"